docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -13,99 +13,192 @@ description: guide for doing a lift and shift migration of airflow to dagster
+
+To install everything, run:
 
 ```bash
-pip install dagster dagster-airflow
+pip install dagster dagster-airflow dagit
 ```
 
+We also suggest verifying that you're installing the correct versions of your Airflow dependencies. Verifying the dependency versions will likely save you from debugging tricky errors later.
+
+To check dependency versions, open your Airflow provider's UI and locate the version numbers. When finished, continue to the next step.
+
 ---
 
-## Step 1: Create a new repository using make_dagster_repo_from_airflow_dags_path
+## Step 3: Convert DAGS into Dagster definitions
 
-The first step to migrating is to define a Dagster repository. To do this, we'll use <PyObject module="dagster_airflow" object="make_dagster_repo_from_airflow_dags_path" /> and pass it the file path of our Airflow Dag Bag. Dagster will then load the DagBag and convert all DAGs into Dagster jobs.
+In this step, you'll start writing Python!
+
+In the `dagster_migration.py` file you created in [Step 1](#step-1-prepare-your-project-for-a-new-dagster-python-module), use <PyObject module="dagster_airflow" object="make_dagster_definitions_from_airflow_dags_path" /> and pass in the file path of your Airflow DagBag. Dagster will load the DagBag and convert all DAGs into Dagster jobs and schedules.
 
 ```python file=/integrations/airflow/migrate_repo.py
 import os
 
 from dagster_airflow import (
-    make_dagster_repo_from_airflow_dags_path,
+    make_dagster_definitions_from_airflow_dags_path,
 )
 
-migrated_airflow_repo = make_dagster_repo_from_airflow_dags_path(
-    os.path.join(os.environ["AIRFLOW_HOME"], "dags"),
-    "migrated_airflow_repo",
+migrated_airflow_definitions = make_dagster_definitions_from_airflow_dags_path(
+    os.path.abspath("./dags/"),
 )
 ```
 
-Under the hood, Dagster is running the exact operator code as you were in Airflow. You will be able to view your normal Airflow `stdout`/`stderr` logs as compute logs in Dagit.
+---
+
+## Step 4: Verify the DAGs are loading
+
+In this step, you'll spin up Dagit, Dagster's web-based UI, and verify that your migrated DAGs are loading. **Note**: Unless the migrated DAGs are very, very simple, it's unlikely they'll execute correctly at this point. That's okay - we'll fix it in a bit. Starting Dagit is the first step in our development loop, allowing you to make a local change, view it in Dagit, and debug any errors.
+
+1. Run the following to start Dagit:
+
+   ```bash
+   dagster dev -f ./migrate_repo.py
+   ```
+
+2. In your browser, navigate to <http://localhost:3001>. You should see a list of Dagster jobs that correspond to the DAGs in your Airflow DagBag.
+
+3. Run one of the simpler jobs, ideally one where you're familiar with the business logic. Note that it's likely to fail due to a configuration or permissions issue.
+
+4. Using `stdout/stderr`, identify and address the cause of the failure.
```

Comment:
❓ Does this refer to the logs? I kept it from the original draft, but it felt odd to me. If it's the logs, we should refer users to them in Dagit and mention that they can view `stdout/stderr` there if they want.

Reply:
I'll change it to logs, but specifically these are the compute logs not our dagster event log. I've seen the difference be a point of confusion for users

Reply:
Understood! I think just telling users where to look is the important thing here. I assumed 'logs' in general because the instructions had me in Dagit at the time.